.eslintrc.js

```diff
@@ -26,6 +26,24 @@ module.exports = {
       }
     },
 
+    {
+      'files': [
+        '*.spec.ts'
+      ],
+      'rules': {
+        '@typescript-eslint/no-require-imports': 'off' // required by Jest to mock the imports
```

Comment:
Should we disable the rule everywhere or disable only the concerned lines ?

Reply:
For me it makes sense in anycase to allow it in all Jest test for mock capability.